File: arm.64/langs/python/src/main.py
import sys
from boot import boot
from trip import trip

def main():
    target_dir = sys.argv[1] if len(sys.argv) > 1 else None
    dist = boot(target_dir)
    if dist:
        trips = trip()
        for d in dist:
            for t in trips:
                d.Pump(t.address.encode("utf-8"), t.payload.encode("utf-8"), t.options.encode("utf-8"))

    print(f"{{\n  \"lang\": \"python\",\n  \"status\": \"ready\",\n  \"engines\": {len(dist)}\n}}")

if __name__ == "__main__":
    main()

File: arm.64/langs/node/src/main.ts
import { boot } from './boot.js';
import { trip } from './trip.js';

const targetDir = process.argv[2];
const dist = boot(targetDir);

if (dist.length > 0) {
  const trips = trip();
  for (const d of dist) {
    for (const t of trips) {
      d.Pump(t.address, t.payload ?? '{}', t.options ?? '{"once":true}');
    }
  }
}

const out = {
  lang: 'node',
  status: 'ready',
  engines: dist.length,
};

console.log(JSON.stringify(out, null, 2));
